src/test/java/hw2/Ex1/TestEx1.java

```diff
@@ -0,0 +1,90 @@
+package hw2.Ex1;
+
+
+import hw2.AbstractBaseTest;
+import org.openqa.selenium.*;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class TestEx1 extends AbstractBaseTest {
+
+
+    @Test
+    public void ex1() {
+
+        driver.get("https://epam.github.io/JDI/index.html");
+
+        // Assert Browser title
+        assertEquals(driver.getTitle().trim(), "Home Page");
+        // -----------------------------------------------------------
+
+        // Assert User name expected "PITER CHAILOVSKII
+        login();
+        WebElement loginedUserName = driver.findElement(By.id("user-name"));
+        assertEquals(loginedUserName.getText().trim(), "PITER CHAILOVSKII");
+        // -----------------------------------------------------------
+
+        // Assert 4 items on the header section "HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"
+        List<WebElement> elements = driver.findElements(By.xpath("//ul[@class = 'uui-navigation nav navbar-nav m-l8']/li/a"));
+        List<String> foundedElements = new ArrayList<>();
```

Comment:
Why do you use full list of the classes in the locator? `//ul[@class = 'uui-navigation nav navbar-nav m-l8']/li/a` the number of classes could be reduced. Any changes in the list of the classes will break your locator